resources/intl_verifications/models/intl_verification.yml

```diff
@@ -34,6 +43,21 @@ allOf:
           * deliverable_missing_info — The address is missing some information, but is most likely deliverable.
           * undeliverable — The address is most likely not deliverable. Some components of the address (such as city or postal code) may have been found.
           * no_match — This address is not deliverable. No matching street could be found within the city or postal code.
+      status:
+        type: string
```

Comment:
@sowmitranalla Same question as above: If the string takes one of a fixed set of values, please make it an enum.